justfile

```diff
@@ -39,7 +39,7 @@ test-superchain: clean-ops
 
 # Unit test all Go code in the validation module, and do not run validation checks themselves
 test-validation: clean-ops
-	TEST_DIRECTORY=./validation/... go run gotest.tools/gotestsum@latest --format testname -- -run '[^TestValidation|^TestPromotion|^TestGenesisAllocs]'
+	TEST_DIRECTORY=./validation/... go run gotest.tools/gotestsum@latest --format testname -- -count=1 -run '[^TestValidation|^TestPromotion|^TestGenesisAllocs]'
```

Comment:
Do we want to commit this?

Reply:
We don't have consistency with that flag: we use it for test-ops but not test-superchain or test-validation. Will remove it for now